### PR TITLE
Add suffixes to typos from "pr" to "pu"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -40253,10 +40401,14 @@ privision->provision
...
 privisions->provisions
 privledge->privilege
+privledged->privileged
+privledges->privileges
 privlege->privilege
 privleged->privileged
 privleges->privileges
 privte->private
+prixies->proxies, pixies, privies,
+prixy->proxy, privy,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:40411]:
```suggestion
prixy->proxy, privy ,pricy,
```
(as a `pricey` variant)

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -40267,10 +40419,16 @@ prmitives->primitives
 prmopting->prompting
 proable->probable
 proably->probably
+probabale->probable
 probabalistic->probabilistic
+probabalities->probabilities
+probabality->probability
 probabaly->probably
+probabe->probable
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:40427]: Any occurrences of `probate` typos?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -40320,7 +40481,12 @@ proble->problem, probe, probably,
...
 probles->problems, probes,
+problim->problem
 problimatic->problematic
+problims->problems
+probllem->problem
+probllematic->problematic
+probllems->problems
 problme->problem
 problmes->problems
 probly->probably
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:40492]:
```suggestion
probly->probably, provably,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -41144,6 +41387,8 @@ pupulated->populated
 pupulates->populates
 pupulating->populating
 pupulation->population
+pupup->popup
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:41390]: Is `pupu` too rare?

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:41390]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**